Enable UFMT on test_nestedtensor.py

YuqingJ opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #128359



cc @albanD

YuqingJ committed:
enable UFMT on test_nestedtensor.py

[ghstack-poisoned]

davidberard98 reviewed (approved):
as long as CI is green, LGTM!

(and fingers crossed for no merge conflicts!)

YuqingJ committed:
Update on "enable UFMT on test_nestedtensor.py"




cc albanD

[ghstack-poisoned]

davidberard98 commented:
fyi I think you can put the `skip-pr-sanity-checks` label on the PR to skip the check for <2k LOC changed

YuqingJ committed:
Update on "auto format test_nestedtensor.py (1/2)"


split it into two PRs since it is more than 2k lines of change


cc albanD

[ghstack-poisoned]

YuqingJ commented:
@pytorchbot merge
